MAUI/DataGrid/grid-events.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Grid Events in MAUI DataGrid control | Syncfusion
+description: Learn here all about Grid Events support in Syncfusion MAUI DataGrid (SfDataGrid) control and more about it.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# Grid Events in MAUI DataGrid (SfDataGrid)
+
+## Cell Tap Events
+
+The `Datagrid` provides the following Events to handle interactions to the cells.
+
+* [CellTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellTapped) : Called when a tap with a cell has occurred.
+* [CellDoubleTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellDoubleTapped) : Called when the user has tapped a cell with a primary button at the same cell twice in quick succession.
+* [CellLongPress](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellLongPress) : Called when a long-press gesture with a primary button has been recognized for a cell.
+* [CellRightTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellRightTapped) : Called when a right-click mouse gesture has been recognized on a cell.
+
+### CellTapped event
+This event will be triggered while tapping a cell in the DataGrid. This event has the [DataGridCellTappedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCellTappedEventArgs.html) as arguments.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                   CellTapped="dataGrid_CellTapped"
+                   ItemsSource="{Binding OrderInfoCollection}" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void dataGrid_CellTapped(object sender,DataGridCellTappedEventArgs e)
```

Comment:
Space should be given between two arguments in a method,

**private void dataGrid_CellTapped(object sender, DataGridCellTappedEventArgs e)**

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/5cb40317-b768-4217-acb7-7d57527fd7de)
Had updated as discussed.